.github/workflows/deno_deploy.yml

```diff
@@ -18,10 +18,10 @@ jobs:
       - uses: denoland/setup-deno@v1
         with:
           deno-version: v1.x
-      
+
       - name: Generate manifest file for dynamic imports
         run: |
-          deno run --allow-read --allow-write routes-manifest-generate.ts
+          deno run --allow-read --allow-write --allow-net routes-manifest-generate.ts
```

Comment:
this is used to generate the routes-manifest might not help with the deploying itself

Reply:
It's in the library documentation.